src/OpenLoco/Entities/EntityManager.cpp

```diff
@@ -13,13 +15,68 @@ namespace OpenLoco::EntityManager
     loco_global<uint16_t[numEntityLists], 0x00525E4C> _listCounts;
     loco_global<Entity[maxEntities], 0x006DB6DC> _entities;
     loco_global<EntityId_t[0x40001], 0x01025A8C> _entitySpatialIndex;
+    loco_global<uint32_t, 0x01025A88> _entitySpatialCount;
     static loco_global<string_id, 0x009C68E6> gGameCommandErrorText;
     constexpr size_t _entitySpatialIndexNull = 0x40000;
 
     // 0x0046FDFD
     void reset()
     {
-        call(0x0046FDFD);
+        std::fill_n(_entities.get(), maxEntities, Entity{});
+        for (auto& count : _listCounts)
+        {
+            count = 0;
+        }
+        for (auto& head : _heads)
+        {
+            head = EntityId::null;
+        }
+        EntityBase* previous = nullptr;
+        EntityId_t id = 0;
+        for (; id < 19800; ++id)
```

Comment:
Any idea what this constant represents?

Reply:
yeah its just how many non money entities there are